CircleSpawn/CircleSpawn/CircleSpawnController.swift

```diff
@@ -1,11 +1,104 @@
 import UIKit
 
-class CircleSpawnController: UIViewController {
-
-	// TODO: Assignment 1
-
+class SpawnedViewSpawnController: UIViewController, UIGestureRecognizerDelegate {
+    
+    
+    private let size: CGFloat = 100
+    
+    //TODO: Move offsets to CircleView
```

Comment:
Don't move offsets to `CircleView`. Offsets to część obsługi gestów, czyli zadania `UIViewController`, a nie `UIView`.